iot_backend_py/settings.py

```diff
@@ -37,13 +38,14 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'dashboard.apps.DashboardConfig'
```

Comment:
Make 2 lists before `INSTALLED_APPS` list with name `CUSTOM_APPS` and `DEFAULT_APPS` and then do `INSTALLED_APPS = DEFAULT_APPS + CUSTOM_APPS. This will be better way to maintain.

Reply:
Done